dashboard/config/scripts/cspu5_u3l13_debugging_multi.multi

```diff
@@ -19,4 +19,4 @@ pJSN1M6T6FmomzeoU/es4nx8BCbtH6aLl4oqKWcDrIHc0Z6skFdqRpHOxiJp
 ltLgKVSnVGBQQp/Ts4rBhhcN2NQfBCoU7NavEYdLTHhaQjGSunC8dIsSgHoL
 tAAwpH40ENqjjO+tGeGcQOH+MkIzNg3zK+LTpCQRFpNVlbcnCQIjJYXlHX5v
 1ZL35nA4P98sNAqKd14QTCRp/xiIQ1k2ew==
-'
+'
```

Comment:
Don't we want the new line at the end of the file?

Reply:
nope! Doesn't make a difference either way